cloud/tests/unittests/multimodal/test_multimodal.py

```diff
@@ -0,0 +1,74 @@
+import tempfile
+import zipfile
+
+from autogluon.cloud import MultiModalCloudPredictor
+
+
+def test_multimodal_tabular_text(test_helper):
+    train_data = 'tabular_text_train.csv'
+    test_data = 'tabular_text_test.csv'
```

Comment:
So we don't have a directory structure, its all flat? Better to have a directory structure IMO. Not a dealbreaker but just mentioning in case these files get crowded in s3.

Reply:
Yea I think you are right. I will improve this in a follow up PR